apps/src/applab/applab.js

```diff
@@ -1584,6 +1670,13 @@ Applab.toggleDesignMode = function(enable) {
 
   var debugArea = document.getElementById('debug-area');
   debugArea.style.display = enable ? 'none' : 'block';
+
+  var children = $('#divApplab').children();
```

Comment:
good catch, but after a discussion with Sarah it sounds like we (1) probably won't allow elements to be nested using design mode (contrary to my TODO which you read), and (2) may have other "incidentally" nested elements (e.g. a div that contains a button and input) where we *might* not want to make the child elements draggable. so I am going to keep the behavior of only making divApplab's immediate children draggable for now.